examples/test_mass_matrix.py

```diff
@@ -1,21 +1,37 @@
+"""
+USAGE
+=====
+
+$ ipython
+
+In [1]: run poisson_2d_mapping.py -t target -n 10 20 -s -c
+
+In [2]: run -i test_mass_matrix.py
+
+"""
+
 from psydac.linalg.solvers      import inverse
 
+globals().update(namespace)
```

Comment:
@yguclu If this file is to be called explicitely after visualize_matrices, then the line `globals().update(namespace)` is redundant.

Reply:
You are correct, this script was originally designed to run after  `visualize_matrices.py`. I think it's a good idea to have it independent of it, though.